content/en/blog/2023/cloud-foundry.md

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Experience Report: Adopting OpenTelemetry for Metrics in Cloud Foundry'
+linkTitle: OpenTelemetry in Cloud Foundry
+date: 2023-10-31
+author: >-
+  [Matthew Kocher](https://github.com/mkocher) (VMware) and [Carson
+  Long](https://github.com/ctlong) (VMware)
+cSpell:ignore: Kocher unscalable
+---
+
+{{% alert color="primary" %}} As part of
+[OpenTelemetry's vision](http://localhost:1313/community/mission/#telemetry-should-be-built-in),
+we believe that high-quality telemetry can be built in to the entire software
+stack. Therefore we are excited, that there are more and more
+[libraries, services and apps, that integrate OpenTelemetry](/ecosystem/integrations/)
+to provide built in observability to their users.\_
+
+_In this and future blog post, we will spot light these integrations to you! If
+you want your integration to be listed and to be highlighted in a blog post as
+well,
+[reach out](https://github.com/open-telemetry/opentelemetry.io/issues/new)!_
+{{% /alert %}}
```

Comment:
This version of the alert has an **invalid link** and the markdown is **misformatted**. I've already applied copyedits to this version as it is found in #3431.

@svrnm @cartermp et al. I'd suggest that we:

- Work on the alert text through #3431 first. (Let's not do it in two PRs at the same time 🙃)
- After #3431 is merged, I can factor the note out into a reusable shortcode
- We can then include the shortcode here.

How does that sound?

Reply:
Ok, let's see what we can do.
Regardless, we'll need a @open-telemetry/blog-approvers to approve.
Maybe first, bring over the copyedits I made to the alert in #3431?

Reply:
Also, you'll need to rebase:

> ⚠️ **This branch is out-of-date with the base branch**

Reply:
@chalin how does that look now with the changes?

Reply:
@ctlong - the badge has landed. **Rebase** to have access to it. For an example of how to add the badge to your post, see `content/en/blog/2023/integrations.md` in:

- https://github.com/open-telemetry/opentelemetry.io/pull/3505/files

Essentially, just add `{{< blog/integration-badge >}}` at the top of your page.